features/steps/contact_step.py

```diff
@@ -0,0 +1,30 @@
+from multiprocessing import context
+from behave import given, then, when
+from features.page_objects.base_page import click_element_by_css_selector, element_displayed, get_alert_text, get_element_text, send_keys_by_css_selector, wait_until_alert_is_present, wait_until_url_changes
+from features.page_objects.contact_page import CONTACT_EMAIL, CONTACT_NAME, MAIL_FIELD, MESSAGE, MESSAGE_FIELD, NAME_FIELD, SEND_MESSAGE_BTN
+from features.page_objects.login_page import LOGIN_BTN, LOGIN_URL
+from features.utils.CommonConstants import CONTACT_BTN, FIELD_PASS, FIELD_USERNAME, LOGIN_BUTTON, LOGOUT_BUTTON, USER_LOG, VALID_PASSWORD, VALID_USERNAME
+from utils.Driver import launch_browser
+import time
+
+@given ('the user is on contact button')
```

Comment:
2 things:
1- the user cant be on contact button, the user can be on the contact form.
but your given should be that you are on the page. and the second action should be to send the message. 
2- this given is a copy of the one you had on login, but you didn't change the function name.
you should not do this. because is repeating code with the same logic of the given that we created to reuse.
if this is a little hard to understand from this text we can talk about it on the class.